script.js

```diff
@@ -0,0 +1,309 @@
+
+const code = [ "Backquote", "Digit1", "Digit2", "Digit3", "Digit4", "Digit5",
+"Digit6", "Digit7", "Digit8", "Digit9", "Digit0", "Minus", "Equal", "Backspace",
+"Tab", "KeyQ", "KeyW", "KeyE", "KeyR", "KeyT", "KeyY", "KeyU", "KeyI", "KeyO",
+"KeyP", "BracketLeft", "BracketRight", "Backslash", "Delete", "CapsLock",
+"KeyA", "KeyS", "KeyD", "KeyF", "KeyG", "KeyH", "KeyJ", "KeyK", "KeyL",
+"Semicolon", "Quote", "Enter", "ShiftLeft", "KeyZ", "KeyX", "KeyC", "KeyV",
+"KeyB", "KeyN", "KeyM", "Comma", "Period", "Slash", "ArrowUp", "ShiftRight",
+"ControlLeft", "MetaLeft", "AltLeft", "Space", "AltRight", "ArrowLeft",
+"ArrowDown", "ArrowRight", "ControlRight" ];  
+ 
+const main = `<main>
+                <div class="wrapper">
+                <textarea class="input" name="" id="input" cols="30" rows="10"></textarea>
+                    <p>Клавиатура создана в операционной системе Windows.</p>
+                    <p>Для переключения языка используйте комбинацию клавиш: Ctrl Left + Alt Left</p>
+              </div>
+              </main>`
+
+let caps_lock = false;
+
+// Add keyboard in html page
+function render_keyboard() {
+    document.querySelector('body').innerHTML = main;
+    let wrapper = document.querySelector(".wrapper");
+    let textarea = document.querySelector(".input");
+    let keyboard = document.createElement("div");
+    keyboard.classList.add("keyboard");
+    wrapper.insertBefore(keyboard, textarea.nextSibling);
+
+    eng_lower.forEach((value, index) => {
+        let button = document.createElement("div");
+        button.classList.add("button");
+        button.classList.add(`${code[index]}`);
+        if (value == "Backspace" || value == "Tab" || value == "Del") {
+          button.classList.add(`${value.toLowerCase()}`);
+          keyboard.appendChild(button);
+        } else if (value == "CapsLock" || index == 42) {
+          button.classList.add("left-btn");
+          keyboard.appendChild(button);
+        } else if (value == "Enter" || index == 54) {
+          button.classList.add("right-btn");
+          button.value = value;
+          keyboard.appendChild(button);
+        } else if (value == " ") {
+          button.classList.add("space");
+          keyboard.appendChild(button);
+        } else keyboard.appendChild(button);
+      });
+}
+
+render_keyboard();
+
+let bord = document.querySelectorAll(".button");
+
+render(rus_lower, eng_lower);
+
+// function add key value for keyboard
+function render_key(language) {
+  language.forEach((key, i) => {
+    bord[i].innerHTML = key;
+  });
+}
+
+function render_key_upper(language) {
+  language.forEach((key, i) => {
+    if (key.length == 1) {
+      bord[i].innerHTML = key.toUpperCase();
+    } else bord[i].innerHTML = key
+  });
+}
+
+
+// function render key value with load html
+function render(rus, eng) {
+  let local = localStorage.getItem("lang");
+  local == "rus" ? render_key(rus) : render_key(eng);
```

Comment:
why "==" comparison instead of "==="

Reply:
Because when I didn’t know, it’s important to write a strict comparison. Fixed, added strict comparison everywhere